Alignment/OfflineValidation/plugins/TrackerGeometryCompare.cc

```diff
@@ -360,7 +361,7 @@ void TrackerGeometryCompare::createROOTGeometry(const edm::EventSetup& iSetup) {
   }
 
   //accessing the initial geometry
-  edm::ESTransientHandle<DDCompactView> cpv;
+  edm::ESTransientHandle<cms::DDCompactView> cpv;
```

Comment:
@adewit @cvuosalo @ianna isn't DDD supposed to coexist for the time being with DDCMS? Here I see a replacement, please correct me in case

Reply:
Fabio is right. We want both. An example of how to handle both old DD and DD4hep in the same file is here:

https://cmssdt.cern.ch/lxr/source/Geometry/MTDGeometryBuilder/plugins/MTDParametersESModule.cc

Reply:
Should be fixed for the plugins now